data/profileSystemLogic.ts

```diff
@@ -0,0 +1,56 @@
+import { Coding } from '../src/mcode';
```

Comment:
Code should not be in the data directory. The tsconfig file does not expect code to exist within `data`.